README.md

```diff
@@ -21,7 +24,7 @@ pip install wavefront-pyformance
 
 The Wavefront Reporters support tagging at the host level. Tags passed to a reporter will be applied to every metric before being sent to Wavefront.
```

Comment:
SHould it be updated for new name instead of "...to Wavefront" with  "... to Operations for applications"

Reply:
cc @Margarita-Staneva, @shavidissa WDYT?

Reply:
Good point!
I think what we have now is ok. The name of the reporter is `WavefrontDirectReporter`. Therefore, calling it Wavefront reporter is fine (similar to how we continue to use Wavefront Proxy)

Reply:
The name "wavefront reporter" is fine. the question is about the last bit of line

Reply:
Ooos sorry! Yes.. we need to replace that.
 
 Suggestion:

The Wavefront Reporters support tagging at the host level. If you pass a tag through a reporter, the reporter tags the metrics before sending the metrics to our service.

Reply:
I am fine with above suggestion. We should probably run it by Mindy also